strongbox-security-api/src/main/java/org/carlspring/strongbox/security/jaas/Roles.java

```diff
@@ -38,4 +38,12 @@ public void setRoles(Set<Role> roles)
         this.roles = roles;
     }
 
+    @Override
+    public String toString()
+    {
+        final StringBuilder sb = new StringBuilder("\n\tRoles{");
+        sb.append("roles=").append(roles);
+        sb.append('}');
+        return sb.toString();
```

Comment:
Another code-style one, could you please always put an empty line above the `return` statement? It won't make the code better, or anything, but it will make it more obvious where the return statement is. (I've fixed most of the other cases).